instances/atlas/parcellationEntity/JBA/JBA_overall.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_overall",
```

Comment:
this seems not correct... do you mean "brain" ?

Reply:
or Julich-Brain ?

Reply:
This is an error.

It seems in our definition there is a brain region called `overall`, (which serves as the root structure). 

do you think it's suitable to remove this instance, and have its children have no parents? 

(taken out of context the last line reads a little dark)

Reply:
I would rename it to JBA_brain and use it as root structure for all GapMaps as well as telencephalon, metencephalon and diencephalon.

Reply:
I doubled checked (https://gin.g-node.org/FZJ-INM1-BDA/20210615-julichbrain-v2.9.0/src/master/workingdir/hbp-d000006_julichbrain-cytoatlas_dev/JulichBrainCytoAtlas/julichbrain-ontology_29.0.json#L1728-L1740)

hartmut seems to use (nested) overal as super structure for the gapmaps (only) and nothing else. (In the json hierarchy there, it was suppose to be a child of cerebral cortex. Why in this case we have overall being parent of itself is a glitch (I use region name to generate id, and output jsonld name. so two regions with the same name meant one will overwrite the other)

I can certainly make adjustments, but I would like a few clarifications:

- I can certainly make a root region called `Julich-Brain` (independent of this `overall` branch node)
- how (if we should at all) change the branch nodes named `overall` (see screengrab below. or you could explore it at https://atlases.ebrains.eu/viewer/#/a:juelich:iav:atlas:v1.0.0:1/t:minds:core:referencespace:v1.0.0:dafcffc5-4826-4bf1-8ff6-46b8a31ff8e2/p:minds:core:parcellationatlas:v1.0.0:94c1125b-b87e-45e4-901c-00daee7f2579-290/@:0.0.0.-W000.._eCwg.2-FUe3._-s_W.2_evlu..7LIx..0.0.0..1LSm)
![Screenshot from 2021-12-09 17-33-59](https://user-images.githubusercontent.com/19381783/145437401-757fd574-6066-4f37-b286-56367f1745dc.png)